app/views/users/index.html.erb

```diff
@@ -7,5 +7,14 @@
 
   <body>
     <h1> This is the Users page</h1>
+    <% @users.each do |user| %>
+      <a href="/users/<%= user.id%>">
+        <div></div>
+        <div>
+          <h2><%= user.name %></h2> 
+          <span>Number Of Posts: <%= user.posts_counter %></span>
+        </div>
+      </a>
+    <% end %>
   </body>
```

Comment:
- [ ] Please make sure that the user info page matches the design, same as the user index page you didn't add the image placeholder also the boxes style around the user info card and the bio box, the button also should match the design to make sure to create a shadow around it make sure that posts are delivered for the user in boxes

| Yours  | design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/216770929-2d8ba29e-faac-4329-9aa3-7deb52d0f706.png)  |![image](https://user-images.githubusercontent.com/74744787/216770879-b0e81ffe-72c3-485c-834c-86296b625410.png)  |

![image](https://user-images.githubusercontent.com/74744787/216771096-f1a20375-3d57-4a03-ba41-a25e0d2e042c.png)